### PR TITLE
As discussed in Issue #103

### PR DESCRIPTION
**Pull request by ReGuess**

### CODE REVIEW

**Comment by ReGuess** [comment]
Something I just realized (& it's late so I can't confirm this) but it might not be necessary for anyone to edit $PATH if we tell Unix-like users to run ./tumblr_backup.py with a dot-slash in front to specify that it is in the current working directory.
https://unix.stackexchange.com/a/397528

File: tumblr_backup.md [hunk 1]
```diff
@@ -19,7 +19,9 @@ You can see an example of its output [on my home page](http://drbeat.li/tumblr).
    [tumblr-utils.zip](https://github.com/bbolli/tumblr-utils/zipball/master)
    or clone the Github repo from `git://github.com/bbolli/tumblr-utils.git`.
 2. Copy or symlink `tumblr_backup.py` to a directory on your `$PATH` like
-   `~/bin` or `/usr/local/bin`.
+   `~/bin` or `/usr/local/bin`. (Or, if you're on Windows, you can just `cd`
+   to the directory with `tumblr_backup.py` in it, since Windows checks the 
+   current working directory before checking `%PATH%`)
```
Comment (bbolli) [inline tumblr_backup.md]: I don't agree with this change. This section is about UNIX-like systems where the path is not changed at all.

Comment (ReGuess) [reply tumblr_backup.md]: So neither Windows nor UNIX-like systems require the $PATH to be changed?

Comment (bbolli) [reply tumblr_backup.md]: If the beginner's document is also adjusted as you would like to, then no.

**Comment by Cebtenzzre** [comment]
Lol.  This should be closed.  The edits to the instructions are misleading, and I don't see how adding normal folders to your PATH is a "security risk" on any platform.

**Pull request closed without merging**